spec/features/users/dashboard_spec.rb

```diff
@@ -1,29 +1,78 @@
 require 'rails_helper'
 
 RSpec.describe "A User's Dashboard" do
-  let(:sam) { User.create(name: 'Sam', email: 'sam@sam.com') }
+  let(:bob) { User.create(name: 'Bob Belcher', email: 'bburger@yahoo.com') }
 
-  it 'displays the name of the user' do
-    visit user_path(sam)
+  describe 'User Info section' do
+    it 'displays the name of the user' do
+      visit "/users/#{bob.id}"
 
-    expect(page).to have_content "Sam's Dashboard"
+      expect(page).to have_content "Bob Belcher's Dashboard"
+    end
   end
 
-  it 'has a button to discover movies' do
-    visit user_path(sam)
+  describe 'discover section' do
+    it 'has a button to discover movies' do
+      visit "/users/#{bob.id}"
 
-    expect(page).to have_button 'Discover Movies'
+      expect(page).to have_button 'Discover Movies'
 
-    click_on 'Discover Movies'
+      click_on 'Discover Movies'
```

Comment:
don't think we need to test presence of button on line 18... test on line 20 shouldn't pass unless button exists